documentation/modules/cruise-control/con-optimization-proposals.adoc

```diff
@@ -6,48 +6,96 @@
 Each optimization proposal is based on the set of xref:con-optimization-goals-{context}[optimization goals] that was used to generate it, subject to any configured xref:capacity-configuration[capacity limits on broker resources].
 
-An optimization proposal is contained in the `Status.Optimization Result` property of a `KafkaRebalance` custom resource. 
-The information provided is a summary of the full optimization proposal. 
-Use the summary to decide whether to:
+All optimization proposals are _dry runs_: you cannot approve a cluster rebalance without first generating an optimization proposal.
+There is no limit to the number of optimization proposals that can be generated.
 
-* Approve the optimization proposal. This instructs Cruise Control to apply the proposal to the Kafka cluster and start a cluster rebalance operation.
+[[contents-optimization-proposals]]
+[discrete]
+== Contents of optimization proposals
 
-* Reject the optimization proposal. You can change the optimization goals and then generate another proposal.
+An optimization proposal comprises a summary and broker load.
 
-All optimization proposals are _dry runs_: you cannot approve a cluster rebalance without first generating an optimization proposal. 
-There is no limit to the number of optimization proposals that can be generated.
+Summary:: The summary provides an overview of the proposed cluster rebalance and indicates the scale of the changes involved.
+A summary of a successfully generated optimization proposal is contained in the `Status.OptimizationResult` property of the `KafkaRebalance` resource.
+The information provided is a summary of the full optimization proposal.
+Broker load:: The broker load shows before and after values for the proposed rebalance, so you can see the impact on each of the brokers in the cluster.
+A broker load is stored in a ConfigMap that contains data as a JSON string.
 
-[discrete]
-== Cached optimization proposal
+== Approving or rejecting an optimization proposal
 
-Cruise Control maintains a _cached optimization proposal_ based on the configured default optimization goals.
-Generated from the workload model, the cached optimization proposal is updated every 15 minutes to reflect the current state of the Kafka cluster.
-If you generate an optimization proposal using the default optimization goals, Cruise Control returns the most recent cached proposal.
+An optimization proposal summary shows the proposed scope of changes.
 
-To change the cached optimization proposal refresh interval, edit the `proposal.expiration.ms` setting in the Cruise Control deployment configuration.
-Consider a shorter interval for fast changing clusters, although this increases the load on the Cruise Control server.
+You can return a summary from the command line using `kubectl describe kafkarebalance _<kafka_rebalance_resource_name>_ -n _<project_name>_`.
```

Comment:
You could add a `-o json | jq <jq query>` command here if you want to jump straight to the summary? I don't know if we go to the level of detail with other describe commands in the docs?

Reply:
We are assuming people have `jq` installed which could be not the case.

Reply:
we suggest the `jq` tool to retrieve the broker load properties. `kubectl get configmap _MY-REBALANCE_ -o json | jq '.["data"]["brokerLoad.json"]|fromjson|.'`  This is the only `jq` mention in the docs. Do we replace or can we add the `jq` option here?  We can something like _If you're using `jq` command line JSON parser tool, you can return the summary using `kubectl get kafkarebalance -o json | jq _<jq_query>_`.

Reply:
@PaulRMellor yeah I would add something like this, to make it clear that `jq` is not mandatory but really useful to extract information in an easier way.